frontend/src/metabase/visualizations/echarts/cartesian/waterfall/model.ts

```diff
@@ -33,9 +58,9 @@ function createDatum({
 export function getWaterfallDataset(
   rows: RowValues[],
   cardColumns: CartesianChartColumns,
-): WaterfallDatum[] {
+): WaterfallDataset {
   const columns = assertMultiMetricColumns(cardColumns);
```

Comment:
I assume this function was introduced in https://github.com/metabase/metabase/pull/36728 but now it shows incorrect diff so I write a nit suggestion here: you can remove typecast from it by changing it to something like:

```
export function assertMultiMetricColumns(
  chartColumns: CartesianChartColumns,
): MultipleMetricsChartColumns {
  if ("breakout" in chartColumns) {
    throw Error("Given `chartColumns` has breakout");
  }

  return chartColumns;
}
```